src/client/common/installer/productInstaller.ts

```diff
@@ -285,17 +286,23 @@ export class DataScienceInstaller extends BaseInstaller {
 @injectable()
 export class ProductInstaller implements IInstaller {
     private readonly productService: IProductService;
+    private interpreterService: IInterpreterService;
 
     constructor(
         @inject(IServiceContainer) private serviceContainer: IServiceContainer,
         @inject(IOutputChannel) @named(STANDARD_OUTPUT_CHANNEL) private outputChannel: OutputChannel
     ) {
         this.productService = serviceContainer.get<IProductService>(IProductService);
+        this.interpreterService = this.serviceContainer.get<IInterpreterService>(IInterpreterService);
     }
 
     // tslint:disable-next-line:no-empty
     public dispose() {}
     public async promptToInstall(product: Product, resource?: InterpreterUri, cancel?: CancellationToken): Promise<InstallerResponse> {
+        const currentInterpreter = await this.interpreterService.getActiveInterpreter(isResource(resource) ? resource : undefined);
```

Comment:
Were we prompting for interpreter even if an interpreter was selected?

Reply:
Never mind. This is for libraries, not the interpreter  itself.

Reply:
```suggestion
        const currentInterpreter = isResource(resource) ? await this.interpreterService.getActiveInterpreter(resource) : resource;
```

The argument is `InterrpeterUri`, it can contain interpreter or a resource.